helm/cluster-cloud-director/values.yaml

```diff
@@ -2,19 +2,15 @@
 
 baseDomain: k8s.test
 connectivity:
-  containerRegistries: {}
```

Comment:
Is there any difference between having an empty value and not having the value here? Will this change behavior? I am not sure.

Reply:
Logically there is no difference. However in the chart/templating it could be that code is not tolerant to an undefined property. This would surface in CI checks, I think.